.github/workflows/test.yml

```diff
@@ -58,7 +58,7 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           name: arduino-create-agent-${{ matrix.operating-system }}
-          path: arduino-create-agent
+          path: arduino-create-agent*
```

Comment:
windows executable ends with `.exe` extension. Leaving it without the * character ends in the CI failing because not finding the file to upload